cmd/kstone-api/app/server.go

```diff
@@ -19,6 +19,7 @@
 package app
 
 import (
+	"flag"
```

Comment:
please add a blank line to group packages. 

    import (
        "io"
        "strconv"

        "golang.org/x/net/context"

        "tkestack.io/kstone/xx"
    )